### PR TITLE
suggest using describe cmd to list pod containers

### PR DESCRIPTION
**Pull request by juanvallejo**
**Release note**:
```release-note
NONE
```

Usability: suggest using `kubectl describe` to obtain available pod containers when running `kubectl attach ...`

cc @soltysh

### CODE REVIEW

**Comment by juanvallejo** [comment]
@soltysh friendly ping

**Review by soltysh — APPROVED** [review]
/lgtm
/ok-to-test
/assign @deads2k 
for approval

**Comment by deads2k** [comment]
@juanvallejo does it tell me which container it is using?

File: pkg/kubectl/cmd/attach.go [hunk 1]
```diff
@@ -311,6 +321,10 @@ func (p *AttachOptions) containerToAttachTo(pod *api.Pod) (*api.Container, error
 		return nil, fmt.Errorf("container not found (%s)", p.ContainerName)
 	}
 
+	if len(p.SuggestedCmdUsage) > 0 {
+		fmt.Fprintf(p.Err, "%s\n", p.SuggestedCmdUsage)
```
Comment (deads2k) [inline pkg/kubectl/cmd/attach.go]: yeah, you need to tell me what you're defaulting to here as is done in exec

**Comment by juanvallejo** [comment]
@deads2k thanks, review comments addressed

**Pull request merged**